### PR TITLE
Coffee & coding

### PR DESCRIPTION
**Pull request by neil-wilkins**
This PR expands the current section on Coffee & Coding talks into its own chapter, with a little more information on what C&C does, how to get in touch, and links to demo code repos and some useful videos in the Microsoft Stream channel.

### CODE REVIEW

File: index.Rmd [hunk 1]
```diff
@@ -29,7 +29,7 @@ To request an R mentor please complete [this mentee form](https://forms.office.c
 This internal training is recommended for those working in Data and Analysis as it enables you to get up and running with R, SQL, Python and GitHub using the MoJ Analytical Platform. The main introductory R, SQL and GitHub sessions are run live in February/March, June/July and October/November each year while you can also work through R, SQL and Python sessions yourself using the training material and/or recordings. To learn more about the sessions currently available and how to access the material and recordings, go to the [Internal Training Group materials](#ITG) chapter.
 
 ## Coffee and Coding and Bitesize sessions
```
Comment (katharinebreeze) [inline index.Rmd:31]: Maybe split this into two headings, one for C&C and one for the bitesize sessions?

File: 04-Coffee-and-Coding.Rmd [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Code can be shared from any language — whether R, Python, SQL, or other more specialised languages.
+
+The format is generally 30–40 minutes for a presentation, followed by time for questions. The presentations usually take the form of a demonstration of a tool or technique and/or a show and tell of work done within the department using these methods.
+
+## Recorded presentations
+
+Where possible, Coffee and Coding presentations are recorded to allow people to catch up on talks they may have missed and to provide a useful training resource for colleagues.
+
+These recordings are available on the [Coffee and Coding Microsoft Stream Channel](https://web.microsoftstream.com/channel/f6aa6c5d-e90c-44b7-8ccc-28a318fa0630). Access should be granted to all new starters, but if you cannot access the videos please contact the Coffee and Coding mailbox.
```
Comment (katharinebreeze) [inline 04-Coffee-and-Coding.Rmd:20]: Just wondering if there are any issues with having these URLs public?

File: index.Rmd [hunk 3]
```diff
@@ -29,7 +29,7 @@ To request an R mentor please complete [this mentee form](https://forms.office.c
...
-The internal training (see above) is complemented by [coffee and coding](https://web.microsoftstream.com/channel/f6aa6c5d-e90c-44b7-8ccc-28a318fa0630) and [bitesize](https://web.microsoftstream.com/channel/5a6012a2-efd6-4b86-902d-98c864427caa) sessions. The contacts are Katharine Breeze and George Papadopoulos respectively.
+The internal training (see above) is complemented by [coffee and coding](#CC) and [bitesize](https://web.microsoftstream.com/channel/5a6012a2-efd6-4b86-902d-98c864427caa) sessions. The contacts are Katharine Breeze and George Papadopoulos respectively.
```
Comment (katharinebreeze) [inline index.Rmd:32]: Please can the contact be our mailbox instead of me?

File: 04-Coffee-and-Coding.Rmd [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+* [An Introduction to Airflow 2.0](https://web.microsoftstream.com/video/712b49b7-8995-4802-95de-c813c8b213bc)
+
+* [Package Management in renv](https://web.microsoftstream.com/video/3ec54ac3-473c-4268-9d54-9f7096338824)
+
+The full catalogue of recorded talks can be found [here](https://web.microsoftstream.com/channel/f6aa6c5d-e90c-44b7-8ccc-28a318fa0630).
+
+## Code repositories
+
+You can find code examples to go along with past Coffee and Coding presentations in our [private repo](https://github.com/moj-analytical-services/Coffee-and-Coding) (only accessible by members of the MoJ Analytical Services GitHub organisation) or [public repo](https://github.com/moj-analytical-services/Coffee-and-Coding-public).
```
Comment (katharinebreeze) [inline 04-Coffee-and-Coding.Rmd:38]: We last updated the public repo two years ago, so maybe we want to remove reference to that or caveat that only a few presentations are available there?

File: 04-Coffee-and-Coding.Rmd (hunk 2 above)
Comment (katharinebreeze) [inline 04-Coffee-and-Coding.Rmd:20]: > Access should be granted to all new starters

Maybe rephrase to:

> We regularly update the access list, but if you cannot access the videos please contact the Coffee and Coding mailbox.

As there's a reasonably high chance new starters won't have access?

File: index.Rmd (hunk 3 above)
Comment (katharinebreeze) [inline index.Rmd:32]: Just fixing the capitalisation -

```suggestion
The internal training (see above) is complemented by [Coffee and Coding](#CC) and [bitesize](https://web.microsoftstream.com/channel/5a6012a2-efd6-4b86-902d-98c864427caa) sessions. The contacts are Katharine Breeze and George Papadopoulos respectively.
```

**Review by katharinebreeze — COMMENTED** [review]
Looks good! I've added a few suggestions and checked the MS stream links work. 👍

File: 04-Coffee-and-Coding.Rmd (hunk 4 above)
Comment (neil-wilkins) [reply 04-Coffee-and-Coding.Rmd:38]: Agreed, removed

```suggestion
You can find code examples to go along with past Coffee and Coding presentations in our [private repo](https://github.com/moj-analytical-services/Coffee-and-Coding) (only accessible by members of the MoJ Analytical Services GitHub organisation).
```

File: 04-Coffee-and-Coding.Rmd (hunk 2 above)
Comment (neil-wilkins) [reply 04-Coffee-and-Coding.Rmd:20]:
```suggestion
These recordings are available on the [Coffee and Coding Microsoft Stream Channel](https://web.microsoftstream.com/channel/f6aa6c5d-e90c-44b7-8ccc-28a318fa0630). We regularly update the access list, but if you cannot access the videos please contact the Coffee and Coding mailbox.
```

Comment (neil-wilkins) [reply 04-Coffee-and-Coding.Rmd:20]: I think access is locked behind an MoJ Exchange log-in

File: 04-Coffee-and-Coding.Rmd [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+## Get involved!
+
+We are always looking for new presentations. If you have an idea for a talk you would like to give, or if you want to showcase a coding project you have been working on, please contact the Coffee and Coding mailbox.
```
[the remark this answers is not in the document]
Comment (neil-wilkins) [reply 04-Coffee-and-Coding.Rmd:42]:
```suggestion
We are always looking for new presentations. If you have an idea for a talk you would like to give, or if you want to showcase a coding project you have been working on, please contact the Coffee and Coding mailbox: Coffee-and-Coding [at] justice.gov.uk.
```

Agreed, I'll put the @ symbol in brackets to avoid having the direct link.

File: index.Rmd (hunk 1 above)
Comment (neil-wilkins) [reply index.Rmd]: Good idea, done!
